ydb/core/tx/schemeshard/schemeshard_build_index__progress.cpp

```diff
@@ -640,7 +640,7 @@ struct TSchemeShard::TIndexBuilder::TTxReply: public TSchemeShard::TIndexBuilder
         case TIndexBuildInfo::EState::Filling:
         {
             // reschedule shard
-            if (buildInfo.InProgressShards.contains(shardIdx)) {
+            if (buildInfo.InProgressShards.erase(shardIdx)) {
```

Comment:
Double erase here.

Reply:
I don't understand where :(

Reply:
Comment related to commit, it was fixed in next commits